partiql-lang/src/main/resources/org/partiql/schemas/graph.isl

```diff
@@ -0,0 +1,71 @@
+$ion_schema_2_0
+
+/** This ISL schema is a specification of an Ion-based file format for defining graphs externally,
+    in order to be brought into PartiQL data model for querying with the PartQL implementation of GPML.
+    This is useful for general reasons of accessing external data, but is particularly necessary
+    for graphs, since PartiQL does not yet define concrete syntax for them.
+
+    The design of the schema closely follows the graph data model description in
+    PartiQL [RFC-0025](https://github.com/partiql/partiql-docs/blob/main/RFCs/0025-graph-data-model.md),
+    with the exception that the data model in this format is not recursive: the payloads at nodes and edges
+    are general Ion.  That is, it is not specified how a fragment of a payload can be itself recognized as a graph.
+
+    A file contains a graph if it contains a single Ion struct of ISL type Graph, as defined here.
+
+    Annotations specified in this schema (graph, node, edge, dir, undir) are for mnemonic purposes only --
+    graph authors are not required to use them and processors should not attribute semantic meaning to their
+    presence or absence.
+*/
+
+type::{
```

Comment:
I am not sure what you mean here...  
Referring to the fact that nodes and edges are abstract entities in the data model and labels, payloads (as well as ends) are given as functions from node/edge sets to some other sets?  

I guess, some other designs for the format are possible, perhaps more closely reflecting the data model formulation (whatever would be the merits of that). 
This is a one I came up with.  It does not give too many opportunities for writing something wrong, if I may brag so :-)

Reply:
> Referring to the fact that nodes and edges are abstract entities in the data model and labels, payloads (as well as ends) are given as functions from node/edge sets to some other sets?
Yes, and not too opinionated about this since it should not represent our actual data model; I think the only merit would be for reader to get a better sense of our model via this representation.